code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -749,6 +749,15 @@
 		/obj/item/stock_parts/cell = 1)
 	needs_anchored = FALSE
 
+/obj/item/circuitboard/machine/nanobed
+	name = "Nanite Bed (Machine Board)"
+	icon_state = "medical"
+	build_path = /obj/machinery/nanobed
+	req_components = list(
+		/obj/item/stock_parts/scanning_module = 1,
+		/obj/item/stock_parts/manipulator = 1,
+		/obj/item/stock_parts/matter_bin = 1)
```

Comment:
could you maybe add effects to these stock parts? Like, increasing the maximum nanite capacity or the nanite regen rate of the nanobed

Reply:
good idea, i'll add part scaling